experiments/interpretation/dutchf3_patch/local/train.py

```diff
@@ -203,40 +206,43 @@ def run(*options, cfg=None, debug=False):
 
     # Logging:
     trainer.add_event_handler(
-        Events.ITERATION_COMPLETED, logging_handlers.log_training_output(log_interval=config.TRAIN.BATCH_SIZE_PER_GPU),
+        Events.ITERATION_COMPLETED, logging_handlers.log_training_output(log_interval=config.PRINT_FREQ),
     )
     trainer.add_event_handler(Events.EPOCH_COMPLETED, logging_handlers.log_lr(optimizer))
 
-    fname = f"metrics_{config_file_name}_{config.TRAIN.MODEL_DIR}.json" if debug else None
-    evaluator.add_event_handler(
-        Events.EPOCH_COMPLETED,
-        logging_handlers.log_metrics(
-            "Validation results",
-            metrics_dict={
-                "nll": "Avg loss :",
-                "pixacc": "Pixelwise Accuracy :",
-                "mca": "Avg Class Accuracy :",
-                "mIoU": "Avg Class IoU :",
-            },
-            fname=fname,
-        ),
-    )
-
     # Tensorboard and Logging:
     trainer.add_event_handler(Events.ITERATION_COMPLETED, tensorboard_handlers.log_training_output(summary_writer))
     trainer.add_event_handler(Events.ITERATION_COMPLETED, tensorboard_handlers.log_validation_output(summary_writer))
 
+    # add specific logger which also triggers printed metrics on test set
```

Comment:
I'm not sure what you mean by this comment. Also, shouldn't it be "train" set not "test"?

Reply:
good catch - not sure why it says test set - changed to training